### PR TITLE
[REVIEW] Fix unsafe string path concatenation

### PR DESCRIPTION
**Pull request by beckernick**
This PR:
- Fixes all remaining unsafe uses of raw string concatenation for paths

Note that this PR changes where the `sentiment_files` directory for the sentiment analysis queries should be placed. It should now be placed inside the data directory itself.

### CODE REVIEW

**Comment by beckernick** [comment]
This is now ready for review, cc @ayushdg @VibhuJawa @Christian8491

File: tpcx_bb/queries/q10/tpcx_bb_query_10_sql.py [hunk 1]
```diff
@@ -88,12 +88,12 @@ def main(data_dir, client, bc, config):
     # These files come from the official TPCx-BB kit
     # We extracted them from bigbenchqueriesmr.jar
     # Need to pass the absolute path for these txt files
-    sentiment_dir = "/".join(config["data_dir"].split("/")[:-3] + ["sentiment_files"])
+    sentiment_dir = os.path.join(config["data_dir"], "sentiment_files")
```
Comment (VibhuJawa) [inline tpcx_bb/queries/q10/tpcx_bb_query_10_sql.py:91]: Just want to make sure this is intentional as we often save files at 2 directory above this.  

So if the path is  `/raid/tpcx-bb/sf1000/parquet_2gb` the sentiment files were previously placed at ``/raid/tpcx-bb/` . This now assumes the same path as `data_dir` which is better but just want to be sure this is intentional.

File: tpcx_bb/queries/q18/tpcx_bb_query_18_sql.py [hunk 2]
```diff
@@ -253,10 +253,10 @@ def main(data_dir, client, bc, config):
     # This txt file comes from the official TPCx-BB kit
     # We extracted it from bigbenchqueriesmr.jar
     # Need to pass the absolute path for this txt file
-    sentiment_dir = "/".join(config["data_dir"].split("/")[:-3] + ["sentiment_files"])
+    sentiment_dir = os.path.join(config["data_dir"], "sentiment_files")
```
Comment (VibhuJawa) [inline tpcx_bb/queries/q18/tpcx_bb_query_18_sql.py:256]: Same confirmation here.

**Review by VibhuJawa — CHANGES_REQUESTED** [review]
This almost looks good to merge to me after clarification about the `sentiment_dir`. 

Thanks a lot for the work on fixing this and preventing annoying issues.

File: tpcx_bb/queries/q10/tpcx_bb_query_10_sql.py (hunk 1 above)
Comment (VibhuJawa) [reply tpcx_bb/queries/q10/tpcx_bb_query_10_sql.py:91]: This change was intentional (see PR description)

**Review by VibhuJawa — APPROVED** [review]
The pr looks good to go after a minor change to `README.md` about the new path of the `sentiment_dir`.

**Comment by beckernick** [comment]
Will factor out the blazingsql create table calls into a helper function to avoid duplication in a followup shortly

**Pull request merged**